fuse/ipns/ipns_unix.go

```diff
@@ -285,7 +285,7 @@ func (s *Node) Attr() fuse.Attr {
 }
 
 // Lookup performs a lookup under this node.
-func (s *Node) Lookup(name string, intr fs.Intr) (fs.Node, fuse.Error) {
+func (s *Node) Lookup(name string, ctx context.Context) (fs.Node, error) {
```

Comment:
@cryptix is this rebased on latest master? I have tests for ipfs fuse that should catch this. Also do the 'static type assertions' that @tv42 suggests.